tests/migrations/helpers.py

```diff
@@ -19,7 +19,7 @@ def makemigrations():
     questioner.input = lambda x: 'y'
 
     os.system('find . -name \*.pyc -delete')
-    if VERSION >= (1, 10):
+    if VERSION >= (1, 11):
```

Comment:
This is not the same as "dropping" support, isn't this more like breaking it? :)

Reply:
You're right, I didn't think about it :(